syntax = "proto3";

//import energy databroker to use TrainRequest
import 'energy_databroker.proto';


//Define the message to hold the status of training
message TrainResponse {
  string status = 1;
}

//Define the training service
service Training {
    rpc trainmodel(TrainRequest) returns (TrainResponse);
}